src/web/mod.rs

```diff
@@ -516,11 +516,25 @@ mod test {
     use crate::test::*;
     use web::{match_version, MatchVersion};
 
+    const DEFAULT_ID: i32 = 0;
+
     fn release(version: &str, db: &TestDatabase) -> i32 {
         db.fake_release().name("foo").version(version).create().unwrap()
     }
-    fn version(v: Option<&str>, db: &TestDatabase) -> MatchVersion {
-        match_version(&db.conn(), "foo", v)
+    fn version(v: Option<&str>, db: &TestDatabase) -> Option<String> {
+        strip_id(match_version(&db.conn(), "foo", v))
+    }
+
+    fn strip_id(v: MatchVersion) -> Option<String> {
```

Comment:
This looks useful, can you make it a method on MatchVersion instead of in test code?